cloudmock/aws/mockec2/dhcpoptions.go

```diff
@@ -147,8 +149,11 @@ func (m *MockEC2) CreateDhcpOptions(request *ec2.CreateDhcpOptionsInput) (*ec2.C
 	}
 	m.DhcpOptions[*dhcpOptions.DhcpOptionsId] = dhcpOptions
 
+	for _, tag := range tags {
+		m.addTag(id, tag)
```

Comment:
Perhaps make `MockEC2.addTag()` be variadic?

Reply:
Thanks, updated